plugins/available/fzf.plugin.bash

```diff
@@ -25,7 +28,7 @@ fe() {
 
 	local IFS=$'\n'
 	local files
-	files=($(fzf-tmux --query="$1" --multi --select-1 --exit-0))
+	IFS=" " read -r -a files --query="$1" --multi --select-1 --exit-0 <<< fzf-tmux
```

Comment:
Is this produced by some tools? I think I saw similar mistakes were introduced and you fixed them before.

Reply:
yeah, I forget to put it in quotes of something, and shfmt gets ideas.